tensorboard/webapp/runs/data_source/runs_data_source_types.ts

```diff
@@ -63,7 +63,7 @@ export interface HparamSpec
   domain: Domain;
 }
 
-export interface MetricSpec extends Omit<backendTypes.MetricSpec, 'name'> {
+export interface MetricSpec extends backendTypes.MetricSpec {
```

Comment:
There are two different definitions of `MetricSpec`. One has a name attribute and one does not. I don't understand why we would want them to be different so I'm ensuring that both have the attribute. In the future I would like to remove this instance of `MetricSpec` and update all references to `MetricSpec.tag` to be `MetricSpec.name.tag`

Reply:
Can we add a TODO for this?

Reply:
And track it with the rest of the outstanding tasks.